packages/functional-tests/tests/misc/resetPasswordKeyStretchingV2.spec.ts

```diff
@@ -0,0 +1,212 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { EmailHeader, EmailType } from '../../lib/email';
+import { expect, test } from '../../lib/fixtures/standard';
+import { BaseTarget } from '../../lib/targets/base';
+
+const AGE_21 = '21';
+const HINT = 'secret key location';
+
+/**
+ * These tests represent various permutations between interacting with V1 and V2
+ * key stretched passwords. We need to ensure that operations are interchangeable!
+ */
+test.describe('severity-2 #smoke', () => {
```

Comment:
The original tests gave an automatic pass for backbone, so this PR removes those tests (see https://github.com/mozilla/fxa/blob/main/packages/functional-tests/tests/misc/keyStretchingV2.spec.ts#L353-L355)